### PR TITLE
Docusaurus documentation site

### PR DESCRIPTION
**Pull request by orcutt989**
This PR creates the initial build pipeline and site structure for a docusaurus-based documentation site for tezos-k8s.

Fix https://github.com/oxheadalpha/tezos-k8s/issues/395

http://www.oxheadalpha.com/tezos-k8s/

### CODE REVIEW

**Review by nicolasochem — CHANGES_REQUESTED** [review]
Lets' not put the default blog post and images there? Let's just have one blank page named "tezos-k8s documentation?"

Also do we want to deploy on release or on push? I think deploying on release makes more sense, but we should deploy on push for now just to see that the pipeline works.

File: .github/workflows/docusaurus.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: docusaurus
+
+on:
+  push:
```
Comment (nicolasochem) [inline .github/workflows/docusaurus.yml:4]: Can we only deploy documentation during releases? We don't want unreleased features to be there.

File: docusaurus/src/pages/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Tezos K8s
```
Comment (nicolasochem) [inline docusaurus/src/pages/index.md:1]: Can this page move to the top level of the repo under `docs/` folder? So it's easier for contributors to find.

**Review by nicolasochem — DISMISSED** [review]

File: .github/workflows/docusaurus.yml (hunk 1 above)
Comment (orcutt989) [reply .github/workflows/docusaurus.yml]: Done

File: docusaurus/src/pages/index.md (hunk 2 above)
Comment (orcutt989) [reply docusaurus/src/pages/index.md]: Done

**Review by nicolasochem — DISMISSED** [review]
thank you

**Review by nicolasochem — COMMENTED** [review]
there are 2 nojekyll files, normal?

**Review by nicolasochem — APPROVED** [review]
thanks!

**Pull request merged**